chsdi/views/layers.py

```diff
@@ -104,9 +104,9 @@ def legend(request):
     return response.body
 
 
-def _find_type(model, colname):
-    if hasattr(model, '__table__') and colname in model.__table__.c:
-        return model.__table__.c[colname].type
+def _find_type(model, colProp):
+    if hasattr(model, '__table__') and hasattr(model, colProp):
+        return model.get_column_by_name(colProp).type
```

Comment:
This method name should be changed since it is confusing. I'll do it in a separate PR.